concourse/pipelines/linux-image-build.yaml

```diff
@@ -1070,6 +1070,22 @@ jobs:
       wf: "debian/debian_11.publish.json"
       environment: "test"
 
+# Image test
+# centos
+- name: image-test-centos-7
+  plan:
+  - get: guest-test-infra
+    passed: [publish-to-testing-centos-7]
```

Comment:
is there a reason we do tests after publishing to testing? just wondering why you picked that

Reply:
Oh. I thought we could use bct-prod-images for testing. that's saving efforts to publish to additional projects.  Otherwise we have to add another publish job and change publish job workflow